docs/data/tree-view/rich-tree-view/editing/items.ts

```diff
@@ -0,0 +1,37 @@
+import { TreeViewBaseItem } from '@mui/x-tree-view/models';
+
+export type ExtendedTreeItemProps = {
+  editable?: boolean;
+  id: string;
+  firstName: string;
+  lastName: string;
+};
+
+export const ITEMS: TreeViewBaseItem<ExtendedTreeItemProps>[] = [
+  {
+    id: '1',
+    firstName: 'Jane',
+    lastName: 'Doe',
+    editable: true,
+    children: [
+      { id: '1.1', firstName: 'Elena', lastName: 'Kim', editable: true },
```

Comment:
Do we need to manually add `editable` everywhere for demos where all the items are editable.

Maybe we can just pass `isItemEditable={() => true}`, it could even be a small `:::success` note in the doc when we introduce this prop to say that you can enable editing for all items easily